Tests/InsightsTests.swift

```diff
@@ -13,6 +13,7 @@ final class InsightsTests: XCTestCase {
     override func setUp() {
         super.setUp()
         EventManager.shared.deleteEvents(before: Date().epochMilliseconds())
+        Insights.clearInstance()
```

Comment:
Add a separate function for clearInstance and remove this line.

    
    func testClearInstance() {
        Insights.setup(environment: environment,
                       programToken: programToken,
                       sdkVersion: sdkVersion,
                       apiUrl: InsightsTestHelper.restURL,
                       userToken: userToken)
        XCTAssertNotNil(Insights.shared, "Shared instance shouldn't equal to Nil")
        Insights.clearInstance()
        XCTAssertNil(Insights.shared, "Insight instance should equal to Nil")
    }